content/docs/concepts/authorization/v1.x/Backup and Restore/helm/_index.md

```diff
@@ -120,4 +120,4 @@ volumes:
     claimName: redis-backup
 ```
 
-Once saved, Redis will now use the backup volume. 
+Once saved, Redis will now use the backup volume.
```

Comment:
Change to read "...Redis now uses the backup volume."

Reply:
addressed